app/controllers/album.js

```diff
@@ -0,0 +1,30 @@
+const albumsService = require('../services/albums'),
+  logger = require('../logger'),
+  message = require('../constants');
+
+exports.getAlbumsInventory = (req, res, next) => {
+  logger.info(`${message.PREVIOUS_MESSAGE} to list of albums`);
+  albumsService
+    .getAlbums()
+    .then(response => {
+      logger.info(message.MESSAGE_OK);
+      res.status(200).send(response);
+    })
+    .catch(next);
+};
+
+exports.getPhotos = (req, res, next) => {
+  const idAlbum = req.params.id;
+  if (idAlbum > 100 || isNaN(idAlbum)) {
+    throw Error('Make sure your request is correct');
+  }
+  logger.info(`${message.PREVIOUS_MESSAGE} to list of images of an album by the id: ${idAlbum}`);
+  albumsService
+    .getPhotosById(idAlbum)
```

Comment:
What pass if tomorrow you need to search photos by name? You can create a function `getPhotosBy` and pass the condition. In this case, the condition is id